src/lang_items.rs

```diff
@@ -72,7 +73,9 @@ unsafe fn report_panic() -> ! {
         if let (Ok(leds_driver), Ok(timer_driver)) = (leds_driver, timer_driver) {
             let _ = blink_all_leds(&leds_driver, &timer_driver).await;
         }
-        loop {}
+        loop {
```

Comment:
For clarity this should land inside the else-branch: `blink_all_leds` loops infinitely and yields (inside the delay). The yield here is only necessary if no led driver is available.